server/config/routes.js

```diff
@@ -132,6 +132,7 @@ exports.configure = function configure(app) {
   app.delete('/accounts/types/:id', accounts.types.remove);
 
   // API for account routes crud
+  app.get('/accounts/document', accounts.document);
```

Comment:
`/accounts/document` is not very descriptive.  It sounds to me like `patients/document`, which are documents attached to patients.

The Chart of Accounts is a report.  I propose you group it under financial reports, `reports/finance/accounts` or `reports/finance/chart_of_accounts`.